rootfs/etc/snmp/snmptrapd.conf

```
@@ -1,3 +1,5 @@
-disableAuthorization yes
-authCommunity log,execute,net @LIBRENMS_SNMP_COMMUNITY@
+disableAuthorization @SNMP_DISABLE_AUTHORIZATION@
+createUser -e 0x@SNMP_ENGINEID@ @SNMP_USER@ @SNMP_AUTH_PROTO@ @SNMP_AUTH@ @SNMP_PRIV_PROTO@ @SNMP_PRIV@
+authUser @SNMP_ACTION@ @SNMP_USER@ @SNMP_SECURITY_LEVEL@
+authCommunity @SNMP_ACTION@ @LIBRENMS_SNMP_COMMUNITY@
```

To align with net-SNMP, `ACTION` should be renamed `PROCESSING_TYPE`

```suggestion
authUser @SNMP_PROCESSING_TYPE@ @SNMP_USER@ @SNMP_SECURITY_LEVEL@
authCommunity @SNMP_PROCESSING_TYPE@ @LIBRENMS_SNMP_COMMUNITY@
```